react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react"
+
+declare module "react-hot-loader" {
+
+  interface ErrorReporterProps {
+    error: any
+  }
+
+  export interface AppContainerProps {
+    children?: React.ReactElement<any>,
```

Comment:
`children` is not optional (the impl will throw if it doesn't get exactly one child)